tests/conftest.py

```diff
@@ -399,10 +399,20 @@ def wrapper(context, undo):
             api = getattr(package, name + "Api")(client)
             operation_method = getattr(api, operation_name)
 
+            def change_keys_to_snake_case(data):
+                if isinstance(data, dict):
+                    result = {}
+                    for key, value in data.items():
+                        result[snake_case(key)] = change_keys_to_snake_case(value)
+                    return result
+                elif isinstance(data, list):
+                    return [change_keys_to_snake_case(item) for item in data]
+                return data
```

Comment:
this should probably be global function and we should find all places where `json.loads` is used

Reply:
I think that a better solution would be to fix the decorator @convert_js_args_to_python_args

Reply:
Sure! Can you meantime move it to the top-level so it can be used in all places where `json.loads` is used?

Reply:
The only `json.loads` call in `conftest` is covered, the other call is in the `src/datadog_api_client/v2/api_client.py`,
I'm not sure if it should be covered too for now, otherwise I guess we can just add it to the `model_utils`